routes/donationRouter.js

```diff
@@ -7,13 +7,34 @@ const donationRouter = express.Router();
 // GET all donations
 donationRouter.get('/', async (req, res) => {
   try {
-    const donation = await db.query('SELECT * FROM donation_tracking');
+    const { itemsLimit, pageNum } = req.query;
+
+    const donation = await db.query(
+      `
+      SELECT *
+      FROM donation_tracking
+      ${itemsLimit ? `LIMIT ${itemsLimit}` : ''}
+      ${pageNum ? `OFFSET ${(pageNum - 1) * itemsLimit}` : ''};`,
+      { itemsLimit, pageNum },
+    );
```

Comment:
instead on modifying an existing route, make a new route in case someone else was using this so you don't break their code. side note: you're both cut